pkg/commands/workload_apply.go

```diff
@@ -234,6 +273,10 @@ Workload configuration options include:
 
 	// Define common flags
 	opts.DefineFlags(ctx, c, cmd)
+	cmd.Flags().StringVar(&opts.UpdateStrategy, cli.StripDash(flags.UpdateStrategyFlagName), "merge", "specify configuration file update strategy (supported strategies: merge, replace)")
```

Comment:
instead of `merge` string, use the const you have declared

Reply:
merge, replace -> use const as well for this

Reply:
ok, changed